src/axes/timeAxis.ts

```diff
@@ -80,115 +80,120 @@ export class Time extends Axis<Date> {
+      ],
+      [
+        { interval: TimeInterval.minute, step: 30, formatter: formatter("%I:%M %p") },
+        { interval: TimeInterval.day, step: 1, formatter: formatter("%B %e, %Y") },
+      ],
+      [
+        { interval: TimeInterval.hour, step: 1, formatter: formatter("%I %p") },
+        { interval: TimeInterval.day, step: 1, formatter: formatter("%B %e, %Y") },
+      ],
+      [
+        { interval: TimeInterval.hour, step: 3, formatter: formatter("%I %p") },
+        { interval: TimeInterval.day, step: 1, formatter: formatter("%B %e, %Y") },
+      ],
+      [
+        { interval: TimeInterval.hour, step: 6, formatter: formatter("%I %p") },
+        { interval: TimeInterval.day, step: 1, formatter: formatter("%B %e, %Y") },
+      ],
+      [
+        { interval: TimeInterval.hour, step: 12, formatter: formatter("%I %p") },
+        { interval: TimeInterval.day, step: 1, formatter: formatter("%B %e, %Y") },
+      ],
+      [
+        { interval: TimeInterval.day, step: 1, formatter: formatter("%a %e") },
+        { interval: TimeInterval.month, step: 1, formatter: formatter("%B %Y") },
+      ],
+      [
+        { interval: TimeInterval.day, step: 1, formatter: formatter("%e") },
+        { interval: TimeInterval.month, step: 1, formatter: formatter("%B %Y") },
+      ],
+      [
+        { interval: TimeInterval.month, step: 1, formatter: formatter("%B") },
+        { interval: TimeInterval.year, step: 1, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.month, step: 1, formatter: formatter("%b") },
+        { interval: TimeInterval.year, step: 1, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.month, step: 3, formatter: formatter("%b") },
+        { interval: TimeInterval.year, step: 1, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.month, step: 6, formatter: formatter("%b") },
+        { interval: TimeInterval.year, step: 1, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 1, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 1, formatter: formatter("%y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 5, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 25, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 50, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 100, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 200, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 500, formatter: formatter("%Y") },
+      ],
+      [
+        { interval: TimeInterval.year, step: 1000, formatter: formatter("%Y") },
+      ],
+    ];
+  }
+
+  private _useUTC: boolean = false;
```

Comment:
I believe this default will actually be overridden with `undefined` if the constructor is invoked without the third parameter.

when the object instance is first created, `this._useUTC` is initialized with `false`, but then the constructor runs, which explicitly overrides `this._useUTC` with the optional argument. You can fix this by simply moving this default to the constructor signature like `(..., useUTC: boolean = false)`. Or better yet, `(..., useUTC: boolean = DEFAULT_USE_UTC)` and put `const DEFAULT_USE_UTC = false;` at the top of this file.

Reply:
Though, since you're already defining the default at a lower level, you could just not initialize `this._useUTC`

Reply:
Given that default values are set at `time` and `tickInterval`, I think it's fine to leave it potentially undefined. I've removed the initialized value for `_useUTC`